adding intro_gravity

siddhism opened the pull request:
Complete port of intro_gravity. changed the reading behaviour of instructions with a button as suggested.

bdoin commented on src/activities/intro_gravity/Intro_gravity.qml:
```diff
@@ -0,0 +1,205 @@
+*   along with this program; if not, see <http://www.gnu.org/licenses/>.
+*/
+import QtQuick 2.2
+import QtQuick.Controls 1.3
```
Use 'import QtQuick.Controls 1.2' to accept older Qt 5.3 which we use on Android

bdoin commented:
When running it I see this log:
SliderStyle.qml:205:13: QML Loader: Binding loop detected for property "y"

When started in window mode, I don't see the 'let's' go button. You must take care your activity runs fine when you resize your window small or large. It is ok if you reset the level when the window is resized.
When I loose and I go to level 2 I am still in crash mode.
At level 2, probably others there is too many asteroids and they are too slow
The image background.svgz must be fully svg, not a png in a svg or we loose the advantage of the svg.

siddhism commented:
1. qt control 1.2 :done.
2. I couldn't figure how to correct it. there is a known bug.
https://bugreports.qt.io/browse/QTBUG-31441

On Thu, Feb 19, 2015 at 6:55 AM, Bruno Coudoin <notifications@github.com>
wrote:

> When running it I see this log:
> SliderStyle.qml:205:13: QML Loader: Binding loop detected for property "y"
>
> When started in window mode, I don't see the 'let's' go button. You must
> take care your activity runs fine when you resize your window small or
> large. It is ok if you reset the level when the window is resized.
> When I loose and I go to level 2 I am still in crash mode.
> At level 2, probably others there is too many asteroids and they are too
> slow
> The image background.svgz must be fully svg, not a png in a svg or we
> loose the advantage of the svg.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bdoin/GCompris-qt/pull/61#issuecomment-74984074>.
>

bdoin commented:
Le 19/02/2015 05:27, siddhesh suthar a écrit :
> 1. qt control 1.2 :done.
> 2. I couldn't figure how to correct it. there is a known bug.
> https://bugreports.qt.io/browse/QTBUG-31441
>
The second issue here is on 'y' while the bug is on 'value', not sure 
this is the same issue as the bug you found.

siddhism commented:
3. Let's go button is now visible. fixed.
4. Loose and go to level 2 crashing problem fixed. everything works fine
now after resizing. fixed arrow's scale size according to background.width
5. background.png converted to .svg by using inkspace. fixed.


On Thu, Feb 19, 2015 at 9:57 AM, Siddhesh Suthar <siddhesh.it@gmail.com>
wrote:

> 1. qt control 1.2 :done.
> 2. I couldn't figure how to correct it. there is a known bug.
> https://bugreports.qt.io/browse/QTBUG-31441
>
> On Thu, Feb 19, 2015 at 6:55 AM, Bruno Coudoin <notifications@github.com>
> wrote:
>
>> When running it I see this log:
>> SliderStyle.qml:205:13: QML Loader: Binding loop detected for property "y"
>>
>> When started in window mode, I don't see the 'let's' go button. You must
>> take care your activity runs fine when you resize your window small or
>> large. It is ok if you reset the level when the window is resized.
>> When I loose and I go to level 2 I am still in crash mode.
>> At level 2, probably others there is too many asteroids and they are too
>> slow
>> The image background.svgz must be fully svg, not a png in a svg or we
>> loose the advantage of the svg.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/bdoin/GCompris-qt/pull/61#issuecomment-74984074>.
>>
>
>

siddhism commented:
yes sir i need help with that issue.binding error are caused due to cyclic
dependencies but i don't see any now. i am pushing changes. everything else
seems to be fixed.

On Thu, Feb 19, 2015 at 12:51 PM, Bruno Coudoin <notifications@github.com>
wrote:

>
> Le 19/02/2015 05:27, siddhesh suthar a écrit :
> > 1. qt control 1.2 :done.
> > 2. I couldn't figure how to correct it. there is a known bug.
> > https://bugreports.qt.io/browse/QTBUG-31441
> >
> The second issue here is on 'y' while the bug is on 'value', not sure
> this is the same issue as the bug you found.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bdoin/GCompris-qt/pull/61#issuecomment-75008539>.
>

hkaelber commented:
Strange one, this loop! Seems to go away if you set the dimensions of your sliders, e.g.

width: 10
height: planetLeft.height

The Slider somehow seems to mess up its dimensions internally otherwise.

siddhism commented:
Increased speed of asteroids. can i destroy the asteroids as soon as they
cross the shuttle.

On Thu, Feb 19, 2015 at 12:59 PM, Siddhesh Suthar <siddhesh.it@gmail.com>
wrote:

> yes sir i need help with that issue.binding error are caused due to cyclic
> dependencies but i don't see any now. i am pushing changes. everything else
> seems to be fixed.
>
> On Thu, Feb 19, 2015 at 12:51 PM, Bruno Coudoin <notifications@github.com>
> wrote:
>
>>
>> Le 19/02/2015 05:27, siddhesh suthar a écrit :
>> > 1. qt control 1.2 :done.
>> > 2. I couldn't figure how to correct it. there is a known bug.
>> > https://bugreports.qt.io/browse/QTBUG-31441
>> >
>> The second issue here is on 'y' while the bug is on 'value', not sure
>> this is the same issue as the bug you found.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/bdoin/GCompris-qt/pull/61#issuecomment-75008539>.
>>
>
>

siddhism commented:
Great. got rid of the loop . thanks a lot.


On Thu, Feb 19, 2015 at 2:27 PM, Holger Kaelberer <notifications@github.com>
wrote:

> Strange one, this loop! Seems to go away if you set the dimensions of your
> sliders, e.g.
>
> width: 10
> height: planetLeft.height
>
> The Slider somehow seems to mess up its dimensions internally otherwise.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bdoin/GCompris-qt/pull/61#issuecomment-75017073>.
>
